homeworks/rostyslav.havuka_RostyslavHavuka/6-typeScript/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+      "outDir": "./dist/",
+      "sourceMap": true,
+      "noImplicitAny": true,
+      "module": "es6",
+      "target": "es5",
+      "jsx": "react",
```

Comment:
You don't really need `jsx` here, I guess. Try to remove and check does everything work. If not, leave as it is